fix: Update progress bar when refreshing after adding new records

leiyre opened the pull request:
closes #1590

this PR forces "Refresh" button to update progress metrics

leiyre committed:
fix: Update progress bar when refreshing after adding new records

closes #1590

this PR forces "Refresh" button to update progress metrics

frascuchon reviewed (changes requested):
Weird behavior validating token classification records. After validate some records, the refresh button reset the counter to the initial state.  See the gif

![compute-wrong-metrics](https://user-images.githubusercontent.com/2518789/185499461-abab5f82-f169-4194-b7ab-ae429163d460.gif)

leiyre committed:
update globalResults when refreshing

globalResults aggregations and total should be updated after clicking the refresh button

leiyre committed:
fetch annotation progress when refreshing

leiyre committed:
remove refresh dataset aggregations when refreshing

leiyre committed:
refresh aggregations when refreshing
